packages/guui/components/Grid/index.js

```diff
@@ -110,49 +109,33 @@ const normaliseProps = (props: BreakpointProps | number): BreakpointProps => {
     return props;
 };
 
-export const Row = ({
-    htmlTag,
-    children,
-}: {
-    htmlTag?: string,
-    children: React$Node,
-}) => {
-    const GridRow = RowStyled.withComponent(htmlTag);
-
-    return <GridRow>{children}</GridRow>;
-};
-
-Row.defaultProps = {
-    htmlTag: 'div',
-};
+export const Row = ({ children }: { children: React.Node }) => (
+    <div className={row}>{children}</div>
+);
```

Comment:
So we're removing the ability to define the element type of `Row`? I'm not sure we use it now anyway so maybe this is OK and we can revisit if we find we need that functionality again.

Reply:
I see we've done the same with `Cols`. I remember the use case which lead to the introduction of `htmlTag` was something like what if we had a `<ul>` which you wanted to be the Row and each `<li>` would be a column. With this implementation this wouldn't be possible.

Reply:
Yeah, you’re right to flag this up, I was just thinking the same thing. I’ll reinstate the feature using React.createElement